### DISCUSSION TITLE
Feature request: Ceph Data

### DISCUSSION THREAD

**Discussion by beccsjb** [dougiteixeira/proxmoxve#116] [Feature request] [answered]
Hi Guys,

Love the integration and for the most part have had very little issues with it. 

It would be nice to see some Ceph data to trigger notifications if a OSD were to fail for example. Is this at all possible?

**Comment by dougiteixeira** [comment]
Hi, I don't use it so it's hard for me to include it.

Can you share the data returned by the API to CEPH Data?

**Comment by mZ738** [comment, marked as answer]
Ceph is in: health with the failing checks as attributes since 5.0.0, and 5.1.0 ([release notes](https://github.com/dougiteixeira/proxmoxve/releases/tag/5.1.0)) adds `Ceph used`, `Ceph total` and `Ceph used percentage` on the cluster device from the same `cluster/ceph/status` response. It needs the optional cluster credentials (`Sys.Audit` on `/`). One honest note: I run no Ceph, so this has only ever been tested against the API's documented shape — if a value looks wrong on a real cluster, `pvesh get /cluster/ceph/status --output-format json` in a new thread is what I would need. Closing as done.
